packages/spruce-skill/server/events/validate-settings.js

```diff
@@ -3,41 +3,45 @@ const { eventError } = require('../lib/errorHandler')
 module.exports = async (ctx, next) => {
 	try {
 		log.debug('**** validate-settings', { event: ctx.event })
-		if (
-			!ctx.event.payload ||
-			!ctx.event.payload.settings ||
-			!ctx.event.payload.userId
-		) {
+		const { auth, event } = ctx
+
+		if (!auth.User || !auth.Organization) {
```

Comment:
Is this file only used for org-level settings?

Reply:
nope, added a location check there.